docs/guides/Screen-Navigation-Prop.md

```diff
@@ -116,6 +116,26 @@ class HomeScreen extends React.Component {
 }
 ```
 
+*Notes regarding key on go back from*
+
+Let see following event history example:
+```... 
+navigation.navigate(SCREEN_KEY_A);
+...
+navigation.navigate(SCREEN_KEY_B);
+...
+navigation.navigate(SCREEN_KEY_C);
+...
+navigation.navigate(SCREEN_KEY_D);
+```
+
+Now you are on *Screen D* and want to go straight back to *screen A* (popping D, C, and B).
```

Comment:
Nit: remove "straight." The sentence conveys the same message with one fewer word :)